src/utils/suggestions.js

```diff
@@ -233,10 +234,7 @@ const getSuggestions = ({
 			currentValue,
 			wordsToShowAfterHighlight,
 		});
-
-		if (predictiveSuggestions.length) {
-			suggestionsList = predictiveSuggestions;
-		}
```

Comment:
@anik-ghosh-au7 Can you explain the reason for removing this block? It simply avoids the predictive suggestions if they are empty.

Reply:
If predictive suggestions are not present then we should display the normal suggestions. cc @siddharthlatest

Reply:
@bietkul ok, I will change it back

Reply:
@bietkul video for what I was saying - https://www.loom.com/share/e904def11e8a469b8e6d1ecd07cb9438